src/main/java/buchen/dictionary/DictionaryFrame.java

```diff
@@ -0,0 +1,52 @@
+package buchen.dictionary;
+
+import com.opencsv.exceptions.CsvValidationException;
+
+import javax.swing.*;
+import java.awt.*;
+import java.io.IOException;
+import java.util.List;
+
+public class DictionaryFrame extends JFrame {
+    private final JTextField wordField;
+    private final JTextArea defArea;
+    private final EnglishDictionary engDic;
+
+    public DictionaryFrame() throws CsvValidationException, IOException {
+        engDic = new EnglishDictionary();
+        setSize(800, 600);
+        setTitle("English Dictionary");
+        setDefaultCloseOperation(EXIT_ON_CLOSE);
+
+        setLayout(new BorderLayout());
+
+        wordField = new JTextField();
+        wordField.setText("Word");
+
+        wordField.getDocument().addDocumentListener(
+                (SimpleDocumentListener) e -> updateDefinition());
+
+
+        defArea = new JTextArea();
+        defArea.setEditable(false);
+        defArea.setLineWrap(true);
+        defArea.setWrapStyleWord(true);
+        JScrollPane scrollPane = new JScrollPane(defArea);
+
+        add(wordField, BorderLayout.NORTH);
+        add(scrollPane, BorderLayout.CENTER);
+
+    }
+
+    private void updateDefinition() {
+        SwingUtilities.invokeLater(() -> {
+            String word = wordField.getText();
+            List<String> defs = engDic.getDefinition(word);
+            defArea.setText("");
+            for (String def : defs) {
+                defArea.append(def + "\n");
```

Comment:
`defArea.append(def).append("\n");`

Reply:
<img width="629" alt="Error" src="https://github.com/AlizaBuchen/buchen-dictionary-2024/assets/121688688/6fbcc02c-64ea-41e9-8100-3f6b23bf3ac1">


When I write it this way It's giving me an error, do you have any advice?

Reply:
oops.
```
defArea.append(def);
defArea.append("\n");
```